qiita_ware/ebi.py

```diff
@@ -404,10 +380,6 @@ def add_sample_prep(self, sample_name, platform, file_type, file_path,
             raise KeyError("Sample %s: sample has not yet been associated "
                            "with the submission.")
 
-        if self.samples[sample_name]['prep']:
-            raise KeyError("Sample %s: multiple rows in prep with this sample "
-                           "id!" % sample_name)
-
```

Comment:
Cause this is impossible if you get the sample names from the DB. I think this check was there for loading from files ...